File: packages/cli/test/fixtures/ranking-corpora/typescript/src/handlers/delete.ts
import { logError, logInfo } from '../util/logger.js';

/** Removes a user by id. */
export function deleteUser(id: number): void {
  if (id <= 0) {
    logError('deleteUser', new Error('id must be positive'));
    return;
  }
  logInfo('deleted user');
}

File: packages/cli/test/fixtures/ranking-corpora/go/internal/handlers/delete.go
package handlers

import (
	"errors"

	"github.com/lien-fixtures/ranking-go/internal/util"
)

// DeleteUser removes a user by id, logging the outcome through the shared
// util package.
func DeleteUser(id int) error {
	if id <= 0 {
		err := errors.New("id must be positive")
		util.LogError("DeleteUser", err)
		return err
	}
	util.LogInfo("deleted user")
	return nil
}
